src/Bridge/Doctrine/Orm/Util/QueryChecker.php

```diff
@@ -20,8 +20,12 @@
  * @author Teoh Han Hui <teohhanhui@gmail.com>
  * @author Vincent Chalamon <vincentchalamon@gmail.com>
  */
-abstract class QueryChecker
+final class QueryChecker
 {
+    private function __construct()
```

Comment:
The constructor is private. It's to block the possibility of initializing this class (it should only conain static methods).